tests/platform_tests/test_reboot.py

```diff
@@ -147,6 +147,14 @@ def check_interfaces_and_services(dut, interfaces, xcvr_skip_list,
         check_sysfs(dut)
 
     if reboot_type is not None:
+        logging.info("Check the determine-reboot-cause service")
+        os_version = dut.os_version.split(".")[0]
+        if os_version < "202106":
```

Comment:
@vvolam why this check required? is it failing on say 202205?

Reply:
This service "determine-reboot-cause" is not present in older versions earlier to 202106.